src/Laravel/config/api-platform.php

```diff
@@ -65,7 +65,14 @@
         'nesting_separator' => '__',
         'introspection' => ['enabled' => true],
         'max_query_complexity' => 500,
-        'max_query_depth' => 200
+        'max_query_depth' => 200,
+        'middleware' => []
```

Comment:
can we maybe comment this or leave to `null`? `middleware` on Laravel takes `null|string|[]`